lib/matplotlib/patches.py

```diff
@@ -4060,6 +4060,20 @@ def __init__(self, posA=None, posB=None,
         Valid kwargs are:
         %(Patch)s
         """
+        if arrow_transmuter is not None:
+            cbook.warn_deprecated(
+                3.0,
+                message=('The "arrow_transmuter" keyword argument is not used,'
+                         ' and will be removed in Matplotlib 3.1'),
```

Comment:
You can do something like `cbook.warn_deprecated("3.0", "arrow_transmuter", obj_type="keyword argument", removal="3.1")` and that'll give you the right message.

Reply:
@dstansby, did you want to do this, or should I just merge as is?

Reply:
If the message looks fine as is then can it be merged as is?